### PR TITLE
feat: side panel header with new design

### PR DESCRIPTION
**Pull request by pspaczek**
:wave: @AlanLes here is my proposition to changes for UiSidePanel header based on right design https://www.figma.com/file/2p8nMM7W88FU6r0kA52fAA/MGP-Component-Library?type=design&node-id=147-16133&mode=dev

### CODE REVIEW

File: src/components/atoms/UiProgress/UiProgress.vue
Comment (pspaczek) [inline src/components/atoms/UiProgress/UiProgress.vue:1]: typo

File: src/components/organisms/UiSidePanel/UiSidePanel.stories.js
Comment (pspaczek) [inline src/components/organisms/UiSidePanel/UiSidePanel.stories.js:1]: stories for "custom cases"; We want to create generic components like side-panel and made a "customization" as stories. You will copy story and have a "customization" like mega-menu, patient education or condition header of side-panel

**Comment by AlanLes** [comment]
Great, do you still work on this or may I merge it? :)

File: src/components/organisms/UiSidePanel/UiSidePanel.vue [hunk 1]
```diff
@@ -360,40 +360,27 @@ onBeforeUnmount(() => {
...
-      @include mixins.use-logical($element + "-tablet-label", padding, 0 var(--space-8));
-    }
-  }
-
-  &__subtitle {
-    @include mixins.use-logical($element + "-subtitle", margin, var(--space-8) 0 0);
+    display: flex;
+    flex-direction: functions.var($element + "-label", flex-direction, column);
+    flex: 1;
+    gap: functions.var($element + "-label", gap, var(--space-4));
```
Comment (AlanLes) [inline src/components/organisms/UiSidePanel/UiSidePanel.vue:383]: Why aren't these properties a css-variables?

File: src/components/organisms/UiSidePanel/UiSidePanel.vue [hunk 2]
```diff
@@ -360,40 +360,27 @@ onBeforeUnmount(() => {
     display: flex;
     flex: none;
     flex-direction: functions.var($element + "-header", flex-direction, row-reverse);
-    justify-content: functions.var($element + "-header", justify-content, space-between);
     background: functions.var($element + "-header", background, var(--color-background-subtle));
-    gap: functions.var($element + "-header", gap, var(--space-32));
+    gap: functions.var($element + "-header", gap, var(--space-16));
```
Comment (AlanLes) [inline src/components/organisms/UiSidePanel/UiSidePanel.vue:364]: Why is there a 16px and not 32?

**Pull request merged**